backend/src/models/user.ts

```diff
@@ -0,0 +1,24 @@
+import mongoose from 'mongoose';
+
+const userScema = new mongoose.Schema({
+  auth0Id: {
+    type: String,
+    required: true,
+  },
+  email: {
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/omkardongre/Food-Ordering-App/pull/14#issuecomment-2363140612>#d9c5f2</a><span> - 09/20/2024, 08:21 am</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The 'email' field lacks validation for email format.
</div>
<div id="fix">
<b>Fix:</b> Add a 'match' validator to the email field using a regular expression for email format validation.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -7,6 +7,8 @@
    },
    email: {
      type: String,
      required: true,
 +    match: [/^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/, 'Please fill a valid email address'],
 +    lowercase: true,
    },
    name: {
      type: String,
```
</div>
</details>
</div>
<br>


</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged

Reply:
good catch, will update this